renku/core/management/storage.py

```diff
@@ -249,3 +254,56 @@ def checkout_paths_from_storage(self, *paths):
             stderr=STDOUT,
             check=True,
         )
+
+    def check_lfs_migrate_info(self, everything=False):
+        """Return list of file groups in history should be in LFS."""
+        ref = ['--everything'] if everything else [
+            '--include-ref', self.repo.active_branch.name
+        ]
+
+        includes = []
+        excludes = []
+        for p in self.renku_lfs_ignore.patterns:
+            if p.regex is None:
+                continue
+
+            pattern = p.pattern.replace(os.linesep, '').replace('\n', '')
```

Comment:
I think you still need to exclude exclamation marks at the beginning of exclude (or was it include?) patterns

Reply:
Does this mean that my original code to exclude everything would work as expected, because these positive patterns would be interpreted correctly anyway 😭?